### PR TITLE
Fix vulnerabilities with 'npm audit fix'

### PR DESCRIPTION
**Pull request by rgrunber**
- y18n 4.0.0 -> 4.0.1
- Update webpack & webpack-cli to update ssri

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -6,22 +6,10 @@
...
-                ],
-                "background": {
-                    "activeOnStart": true,
-                    "beginsPattern": "Compilation \\w+ starting…",
-                    "endsPattern": "Compilation\\s+finished"
-                }
-            },
+            "problemMatcher": [
+                "$ts-webpack-watch",
+                "$tslint-webpack-watch"
```
Comment (fbricon) [inline .vscode/tasks.json:11]: this requires the eamodio.tsl-problem-matcher extension. See https://github.com/redhat-developer/vscode-commons/blob/master/.vscode/extensions.json#L6

**Comment by rgrunber** [comment]
I think the issue is that webpack-cli update introduced the colorette 1.2.2 requirements, which apparently [doesn't work on node v13](https://github.com/jorgebucaran/colorette/issues/48), which is what our macos builder is using.

File: .vscode/tasks.json (hunk 1 above)
Comment (rgrunber) [reply .vscode/tasks.json:11]: Done.

File: .travis.yml [hunk 2]
```diff
@@ -19,6 +19,9 @@ before_install:
 - if [ $TRAVIS_OS_NAME == "linux" ]; then
       export CXX="g++-4.9" CC="gcc-4.9";
   fi
+- if [ $TRAVIS_OS_NAME == "osx" ]; then
+      nvm install 12.12;
```
Comment (fbricon) [inline .travis.yml:23]: I'd use the same node version consistently across platform builds

Comment (rgrunber) [reply .travis.yml:23]: I noticed [vscode-microprofile](https://github.com/redhat-developer/vscode-microprofile/blob/master/.travis.yml#L25) and [vscode-quarkus](https://github.com/redhat-developer/vscode-quarkus/blob/master/.travis.yml#L25) were doing the same thing for osx, so went with that. I'll eliminate the conditional here and likely those other places as well.

Comment (rgrunber) [reply .travis.yml]: Done.

**Pull request merged**